src/components/Onboarding/UserNameInput.tsx

```diff
@@ -0,0 +1,37 @@
+import { TextInput } from "@mantine/core";
+import { UseFormReturnType } from "@mantine/form";
+
+type UserNameInputProps = {
+    form: UseFormReturnType<
+        {
+            username: string;
+            nickname: string;
+        },
+        (values: { username: string; nickname: string }) => {
+            username: string;
+            nickname: string;
+        }
+    >;
+    inputValueName: string;
+};
+
+export default function UserNameInput({
+    form,
+    inputValueName,
+}: UserNameInputProps) {
+    return (
+        <TextInput
+            label="使用者名稱"
+            description="您在平台上的 ID；請使用英文字母、數字或半形句點、底線與減號"
+            required
+            {...form.getInputProps(inputValueName)}
+            key={form.key(inputValueName)}
+            classNames={{
```

Comment:
The styling follows Mantine's xs size

Reply:
I've tried to apply size="sm" to the TextInput, but the text in the label and description appears to be "really" small. I think "md" fits the Figma design more.